### PR TITLE
Delete history button and progress view

### PR DESCRIPTION
**Pull request by ilyageekhub**

### CODE REVIEW

File: PageSpeed/History - PageSpeed/PageSpeedHistory.storyboard [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="15702" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="16096" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
```
Comment (nedopaka) [inline PageSpeed/History - PageSpeed/PageSpeedHistory.storyboard]: @ilyageekhub these changes should be omitted.

File: PageSpeed/History - PageSpeed/PageSpeedHistory.storyboard [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
     <device id="retina6_1" orientation="portrait" appearance="light"/>
     <dependencies>
-        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="15704"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="16086"/>
```
Comment (nedopaka) [inline PageSpeed/History - PageSpeed/PageSpeedHistory.storyboard]: @ilyageekhub these changes should be omitted.

File: PageSpeed/History - PageSpeed/PageSpeedHistory.storyboard [hunk 3]
```diff
@@ -802,7 +802,7 @@
         </scene>
     </scenes>
     <resources>
-        <image name="info.circle" catalog="system" width="64" height="60"/>
+        <image name="info.circle" catalog="system" width="128" height="121"/>
```
Comment (nedopaka) [inline PageSpeed/History - PageSpeed/PageSpeedHistory.storyboard]: @ilyageekhub these changes should be omitted.

File: PageSpeed/MainViewController.swift [hunk 4]
```diff
@@ -21,8 +21,10 @@ class MainTabBarController: UITabBarController {
             as? NewTestViewController
         let historyViewController = UIStoryboard(name: "Stage-B", bundle: nil)
         .instantiateViewController(identifier: "HistoryTableViewController")
-        let supportViewController = UIViewController()
-        let settingsViewController = UIViewController()
+        let supportViewController = UIStoryboard(name: "Stage-B", bundle: nil)
+        .instantiateViewController(identifier: "SupportViewController")
```
Comment (nedopaka) [inline PageSpeed/MainViewController.swift]: @ilyageekhub here is indentation error.

File: PageSpeed/MainViewController.swift [hunk 5]
```diff
@@ -21,8 +21,10 @@ class MainTabBarController: UITabBarController {
...
+        let settingsViewController = UIStoryboard(name: "Stage-B", bundle: nil)
+        .instantiateViewController(identifier: "SettingsViewController")
```
Comment (nedopaka) [inline PageSpeed/MainViewController.swift]: @ilyageekhub here is indentation error.

File: PageSpeed/Test/NewTestViewController.swift [hunk 6]
```diff
@@ -25,6 +25,7 @@ class NewTestViewController: UIViewController {
     var mobilePageSpeedResult: PageSpeedResponse?
     var desktopPageSpeedResult: PageSpeedResponse?
     var gtMetrixResponse: GTMetrixResponseItem?
+    var progressView: ProgressView?
```
Comment (nedopaka) [inline PageSpeed/Test/NewTestViewController.swift]: @ilyageekhub why **ProgressView** is optional? Also, why you declare global variable that's only used in **processRequestsToServices**? If so, you should remove this global variable and use only local one.

File: PageSpeed/Test/NewTestViewController.swift [hunk 7]
```diff
@@ -78,6 +79,8 @@ class NewTestViewController: UIViewController {
 
     // MARK: - Methods
     func processRequestsToServices(url: String) {
+        progressView = ProgressView()
+        view.addSubview(progressView!)
```
Comment (nedopaka) [inline PageSpeed/Test/NewTestViewController.swift]: @ilyageekhub you should use local variable for pre-loader and initiate it right before
>DispatchQueue.global(qos: .userInitiated).async {

These lines should be removed.

File: PageSpeed/Test/NewTestViewController.swift [hunk 8]
```diff
@@ -110,6 +113,10 @@ class NewTestViewController: UIViewController {
                 testResultsViewController?.servicesEnabledArr = self.servicesEnabledArr
                 testResultsViewController?.gtMetrixResponse = self.gtMetrixResponse
                 self.navigationController?.pushViewController(testResultsViewController!, animated: true)
+                if let progressView = self.progressView{
+                    progressView.removeFromSuperview()
+                    self.progressView = nil
+                }
```
Comment (nedopaka) [inline PageSpeed/Test/NewTestViewController.swift]: @ilyageekhub remove this condition. Local variable for pre-loader should be non-optional.

**Review by nedopaka — COMMENTED** [review]
@ilyageekhub fix errors below.

File: PageSpeed/MainViewController.swift (hunk 4 above)
Comment (ilyageekhub) [reply PageSpeed/MainViewController.swift]: fix

File: PageSpeed/MainViewController.swift (hunk 5 above)
Comment (ilyageekhub) [reply PageSpeed/MainViewController.swift]: fix

File: PageSpeed/Test/NewTestViewController.swift (hunk 6 above)
Comment (ilyageekhub) [reply PageSpeed/Test/NewTestViewController.swift]: fix

File: PageSpeed/Test/NewTestViewController.swift (hunk 7 above)
Comment (ilyageekhub) [reply PageSpeed/Test/NewTestViewController.swift]: fix

File: PageSpeed/Test/NewTestViewController.swift (hunk 8 above)
Comment (ilyageekhub) [reply PageSpeed/Test/NewTestViewController.swift]: fix

File: PageSpeed/MainViewController.swift [hunk 9]
```diff
@@ -12,23 +12,23 @@ import Moya
 // MARK: - MainTabBarController
 
 class MainTabBarController: UITabBarController {
-
+
```
Comment (nedopaka) [inline PageSpeed/MainViewController.swift]: @ilyageekhub here is `Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)`

File: PageSpeed/MainViewController.swift [hunk 10]
```diff
@@ -39,18 +39,18 @@ class MainTabBarController: UITabBarController {
 // MARK: - MainViewController
 
 class MainViewController: UIViewController {
-
+
```
Comment (nedopaka) [inline PageSpeed/MainViewController.swift]: @ilyageekhub here is `Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)`

File: PageSpeed/MainViewController.swift [hunk 11]
```diff
@@ -39,18 +39,18 @@ class MainTabBarController: UITabBarController {
 // MARK: - MainViewController
 
 class MainViewController: UIViewController {
-
+    
     override func viewDidLoad() {
         super.viewDidLoad()
     }
-
+
```
Comment (nedopaka) [inline PageSpeed/MainViewController.swift]: @ilyageekhub here is `Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)`

**Review by nedopaka — COMMENTED** [review]
@ilyageekhub fix errors below.

File: PageSpeed/MainViewController.swift (hunk 10 above)
Comment (ilyageekhub) [reply PageSpeed/MainViewController.swift]: fix

File: PageSpeed/MainViewController.swift (hunk 11 above)
Comment (ilyageekhub) [reply PageSpeed/MainViewController.swift]: fix

File: PageSpeed/MainViewController.swift (hunk 9 above)
Comment (ilyageekhub) [reply PageSpeed/MainViewController.swift]: fix

**Pull request merged**